src/aioftp/client.py

```diff
@@ -658,6 +718,9 @@ async def login(
 
         :raises aioftp.StatusCodeError: if unknown code received
         """
+        if self.explicit_tls:
+            await self.upgrade_to_tls()
```

Comment:
Not sure we need this done like. Why it is «explicit» if it is actually «implicit» right now? Lets leave this on user shoulders to call `upgrade_to_tls` if he wants.